applications/content/widget/CommonScreens.xml

```diff
@@ -33,6 +33,7 @@ under the License.
                 <set field="layoutSettings.companyName" from-field="uiLabelMap.ContentCompanyName" global="true"/>
                 <set field="layoutSettings.companySubtitle" from-field="uiLabelMap.ContentCompanySubtitle" global="true"/>
                 <set field="layoutSettings.styleSheets[]" value="/contentimages/css/contentForum.css" global="true"/>
+                <set field="layoutSettings.styleSheets[]" value="/contentimages/css/contentForum.css" global="true"/>
```

Comment:
Hello @bjugl 
Thanks for your contribution!
It seems you missed to updated the css path.

Reply:
This seems wired, I will check.

Reply:
That seems to be a remnant of an early trial and error phase, that I did not clean up properly, sry.
I decided, to not put them here, but rather into "themes/common-theme/widget/CommonScreens.xml" in "GlobalActions". This way, everything is neatly in common-theme, apart from the WebSiteCMSNav.ftl. I would just clean up this extra line, or do you suggest another way of implementation?

Reply:
I think adding to common-theme seems to be the right approach, you have taken. Thanks for the explanation!